light-base/examples/basic.rs

```diff
@@ -81,32 +79,70 @@ fn main() {
             user_data: (),
         })
         .unwrap();
-
-    // The chain is now properly initialized.
+    // Connection to Polkadot is now properly initialized.
```

Comment:
```suggestion

    // The Polkadot chain is now properly initialized.
```

There's no "connection" at this point yet, so "connection" is confusing.